docs/source/admin/traffic_ops.rst

```diff
@@ -0,0 +1,706 @@
+
+	.. warning:: While relative paths are allowed, they are discouraged, as the path will be relative to the working directory of the `traffic_ops_golang`_ process itself, not relative to the ``cdn.conf`` configuration file, which can be confusing.
+
+:portal: This section provides information regarding a connected UI with which users interact, so that emails can include links to it.
+
+	:base_url: This URL should be the root and/or landing page of the UI. For Traffic Portal instances, this should include the fragment part of the URL, e.g. ``https://trafficportal.infra.ciab.test/#!/``.
+	:docs_url: The actual use of this URL is unknown, but supposedly it ought to point to the documentation for the Traffic Control instance. It's hard to imagine a fantastic reason this shouldn't just always be https://traffic-control-cdn.readthedocs.io
+	:email_from: Most emails sent from the Traffic Ops server will use ``to.email_from``, but specifically password reset requests (which contain a link to a fragment under ``portal.base_url``) will instead use this as the value of their :mailheader:`From` field.
+	:pass_reset_path: A path to be added to ``base_url`` that is the URL of the UI's password reset interface. For Traffic Portal instances, this should always be set to "user".
+	:user_register_path: A path to be added to ``base_url`` that is the URL of the UI's new user registration interface. For Traffic Portal instances, this should always be set to "user".
+
+:riak_conf_path: An optional absolute or relative path to `riak.conf`_. If this field is not defined, is ``null``, or is an empty string (``""``), Traffic Ops will not be able to connect to Traffic Vault.
+
+	.. caution:: If Traffic Ops is unable to connect to Traffic Vault, many of its core features will not function. In particular, it will be impossible to create :term:`Delivery Services` that use HTTPS or DNSSEC.
+
+	.. impl-detail:: The name of this field is derived from the current database used in the implementation of Traffic Vault - `Riak KV <https://riak.com/products/riak-kv/index.html>`_.
+
+:secrets: This is an array of strings, which cannot be empty. The first secret in the array is used to encrypt Traffic Ops authentication cookies - multiple Traffic Ops instances serving the same CDN need to share secrets in order for users logged into one to be able to use their cookie as authentication with other instances.
+:smtp:    This optional section contains options for connecting to and authenticating with an :abbr:`SMTP (Simple Mail Transfer Protocol)` server for sending emails. If this section is undefined (or if ``enabled`` is explicitly ``false``), Traffic Ops will not be able to send emails and certain :ref:`to-api` endpoints that depend on that functionality will fail to operate.
```

Comment:
wasnt this added for `.. versionadded:: 4.0` like the influxdb configuration?

Reply:
yes, yes it was